server/bleep/src/ctags.rs

```diff
@@ -24,6 +24,30 @@ use tracing::{debug, warn};
 pub static CTAGS_BINARY: OnceCell<PathBuf> = OnceCell::new();
 pub type SymbolMap = HashMap<PathBuf, Vec<Symbol>>;
 
+// Extract symbols using Ctags for all languages which are not covered by a more
```

Comment:
as the comment suggests, this might be created dynamically in the future, const-ness might not be an option then. alright to keep for now though i think.